files/default/sql/ddl/2.0.0__initial_tables.sql

```diff
@@ -1901,6 +1902,7 @@ CREATE TABLE IF NOT EXISTS `cached_feature_group` (
   `offline_feature_group`          BIGINT(20)      NOT NULL,
   `online_enabled`                 TINYINT(1)      NULL,
   `default_storage`                TINYINT         NULL,
+  `timetravel_format`              ENUM('HUDI', 'NONE') NOT NULL DEFAULT 'HUDI',
```

Comment:
This should be an `INT`. The persistence in hopsworks-ee takes care of doing the conversion.